app/services/stepfunctions/StateMachineLookup.scala

```diff
@@ -11,26 +11,25 @@ import scala.concurrent.ExecutionContext
 import cats.syntax.either._
 
 object StateMachineLookup {
-  def findMachine(items: List[StateMachineListItem], stateMachinePrefix: String, stage: String): Either[StateMachineError, StateMachine] = {
+  def findMachine(items: List[StateMachineListItem], stateMachinePrefix: String): Either[StateMachineError, StateMachine] = {
```

Comment:
I refactored so that the stateMachinePrefix already includes the stage at this point